### PR TITLE
fix: custom floating menu implementation in notebooks

### PR DESCRIPTION
**Pull request by daibhin**
## Problem
The insertion suggestions popover often conflicted with the actions because of changes in the node height that did not trigger a re-render in the Tippy popover provided in the `FloatingMenu`

https://github.com/PostHog/posthog/assets/6685876/c6cb96d8-e800-41a9-b942-6b5d469bf322

## Changes

Move to a custom implementation that also listens to the height of the editor dom and reacts to updates

https://github.com/PostHog/posthog/assets/6685876/caebdf1c-789b-45f9-a123-422ccecb620b

## How did you test this code?

Clicking about

### CODE REVIEW

File: frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx [hunk 1]
```diff
@@ -1,52 +1,66 @@
...
-                }
-
-                return false
-            }}
-        >
-            <div className="FloatingSuggestion flex items-center justify-content">
-                {Component && <Component previousNode={previousNode} editor={notebookEditor} />}
-            </div>
-        </FloatingMenu>
+        <div className="NotebookFloatingButton absolute" style={{ zIndex: 9999, top: position.top, left: 0 }}>
```
Comment (benjackwhite) [inline frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx:58]: I'm nit-picking but the zIndex should be a proper var, maybe the same as the popover z-index

**Review by benjackwhite — CHANGES_REQUESTED** [review]
One small change, but otherwise this is great and works exactly as described.

File: frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx (hunk 1 above)
Comment (daibhin) [reply frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx]: No, nit picking is good. I'd hate if I was someone else who came across it in future

Comment (daibhin) [reply frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx]: Fixed this up. Anything else you want to see changed?

**Review by benjackwhite — APPROVED** [review]

**Pull request merged**